index.bs

```diff
@@ -1350,6 +1376,42 @@ To <dfn for="XRView">obtain the projection matrix</dfn> for a given {{XRView}} |
 </div>
 
 
+Primary and Secondary Views {#primary-and-secondary-views}
```

Comment:
You appear to have a dash of #1083 in your #1086. 😉

Reply:
Hmm, something very weird happened here, but I fixed it